opus/application/static_media/js/widgets.js

```diff
@@ -591,6 +601,7 @@ var o_widgets = {
 
         delete opus.extras[`qtype-${slugNoNum}`];
         delete opus.extras[`z-${slugNoNum}`];
+        delete opus.extras[`unit-${slugNoNum}`];
```

Comment:
Let's get rid of that line that says "z-". That's left over from some ancient feature we no longer support. Perhaps it was the old version of units? Before my time.